CDF Elected Positions.md

```diff
@@ -1,137 +1,152 @@
 
-Current Representative: Justin Abrahms, eBay
+Current Representative: Ole Lensmar, Testkube
 
 
-## General Member Representative 2
+### General Member Representative 2
 
-Term: 1 year 2/1/2022-2/1/2023
+Term: 1 year 2/14/2023-2/14/2024
 
 Current Representative: Gerard Mcmahon, Fidelity Investments
 
 
-## Committer/Contributor Representative 1
+### Committer/Contributor Representative 1
 
-Term: 1year 7/1/2022-6/30/2023
+Term: 1 year 7/1/2022-6/30/2023
 
-Current Representative: Brad McCoy, 
+Current Representative: Brad McCoy, Basiq
 
 
-## Committer/Contributor Representative 2
+### Committer/Contributor Representative 2
 
 Term: 1 year 7/1/2022-6/30/2023
 
-Current Representative: Andrea Fritoli, 
+Current Representative: Andrea Fritoli, IBM
 
 
-## Treasurer/ Budget Committee Chair
+### Treasurer/ Budget Committee Chair
 
 Term: 1 year
 
 Current Representative: N/A
 
 
-## TOC Chair
+## Technical Oversight Committee
+
+### Chair
 
-Term: 1 year 8/31/2021-8/31/2022
+Term: 1 year 1/17/2023-1/16/2024
 
-Current Representative: Oleg Naneshev, Dynatrace
+Current Representative: Andrea Fritoli, IBM
 
 
-## Project Representative 1
+### Project Representative 1
 
-Term: (2 years) 7/1/2021-6/30/2023
+Term: (2 years) 7/1/2023-6/30/2025
 
-Current Representative: Oleg Nenashev, Dynatrace (Jenkins/Keptn)
+Current Representative: Alan Dong, Yahoo (Screwdriver)
 
 
-## Project Representative 2
+### Project Representative 2
 
-Term: (2 years) 7/1/2021-6/30/2023
+Term: (2 years) 7/1/2023-6/30/2025
 
-Current Representative: Jithin Emmanuel, Yahoo (Screwdriver)
+Current Representative: Andrea Frittoli, IBM (Tekton)
 
 
-## Project Representative 3
+### Project Representative 3
 
-Term: (2 years) 7/1/2021-6/30/2023
+Term: (2 years) 7/1/2023-6/30/2025
 
-Current Representative: Andrea Frittoli, IBM (Tekton)
+Current Representative: Mark Waite, CloudBees (Jenkins)
 
 
-## Project Representative 4
+### Project Representative 4
 
-Term: (2 years) 7/1/2021-6/30/2023
+Term: (2 years) 7/1/2023-6/30/2025
 
 Current Representative: Steve Taylor, DeployHub (Ortelius)
 
 
-## End User Representative 1
+### End User Representative 1
 
-Term: (1 year) 5/1/2022-5/1/2023
+Term: (1 year) 5/1/2023-5/1/2024
 
-Current Representative: Justin Abrams, eBay
+Current Representative: Dadisi Sanyika, Apple
 
 
-## End User Representative 2
+### End User Representative 2
 
-Term: (1 year) 5/1/2022-5/1/2023
+Term: (1 year) 5/1/2023-5/1/2024
 
-Current Representative: Emil Bäckmark, Ericsson
+Current Representative: Neil McGonigle, Fidelity Investments
 
 
-## TOC GB Representative 1
+### GB Representative 1
 
 Term: (2 years) 7/1/2022-7/1/2024
 
 Current Representative: Melissa McKay, JFrog
 
 
-## TOC GB Representative 2
+### GB Representative 2
 
 Term: (2 years) 7/1/2022-7/1/2024
 
 Current Representative: Tracy Ragan, Deploy Hub
 
 
-## TOC GB Representative 3
+### GB Representative 3
 
 Term: (2 years) 7/1/2022-7/1/2024
 
 Current Representative: Robert Reeves, Liquibase
```

Comment:
Robert's company maybe needs changed..??

Reply:
Yeah...maybe. I'm still at Liquibase as an advisor. Meet with the team regularly. But, spending the biggest amount of time at LF.

Really up to the team. I'd like the TOC to reach a lazy consensus and just make the call. I can go either way.

Reply:
This can perhaps be discussed by the TOC as needed and updates can then be reflected to the file via a separate PR. Resolving the issue and will merge the PR.